wordle-web/app.vue

```diff
@@ -3,44 +3,79 @@
     <v-app>
       <v-app-bar color="primary" :elevation="2">
         <template v-slot:prepend>
-          <v-icon color="secondary"> mdi-book </v-icon>
+          <v-btn icon @click="router.push('/')">
+            <v-icon color="#FFFFFF">mdi-wizard-hat</v-icon> <!-- Wizard hat icon -->
+          </v-btn>
+          <v-btn @click="router.push('/')">
+            Welcome to LexiQuest
+          </v-btn>
```

Comment:
Making the icon and the title two separate buttons with hover-over effects is a bit confusing when they go to the same place. Maybe they could both be put in a single button?